ord_schema/proto/reaction.proto

```diff
@@ -861,26 +852,104 @@ message ReactionProduct {
-  enum SelectivityType {
+message ProductMeasurement {
+  // The key of the analysis used to calculate this product-specific
+  // measurement; this should cross-reference the outcome.analyses map.
+  string analysis_key = 1;
+  // The type of measurement. While each measurement is associated with a
+  // specific product compound and is cross-referenced to an analysis through
+  // the analysis_key, there is still ambiguity as to what a measurement
+  // truly represents. For example, an LCMS analysis can lead to a quantitative
+  // YIELD if an authentic standard and internal standard are used to obtain
+  // a calibration curve; sometimes, it is only used to obtain an uncalibrated
+  // "yield" that is better described as an peak AREA or integrated EIC
+  // COUNTS. Understanding the fidelity of the approach taken to measure
+  // the performance of a reaction is important for downstream learning tasks.
+  // Other measurement types are relatively unambiguous (e.g., a melting point
+  // measurement results in a Temperature) but should be specified for
+  // uniformity.
+  enum MeasurementType {
     UNSPECIFIED = 0;
     CUSTOM = 1;
-    EE = 2;  // Enantiomeric excess as a percentage (desired - undesired).
-    ER = 3;  // Enantiomeric ratio. (x:1) (desired / undesired).
-    DR = 4;  // Diasteromeric ratio. (x:1) (desired / undesired).
-    EZ = 5;  // Alkene geometry ratio. (x:1) (E / Z)
-    ZE = 6;  // Alkene geometry ratio. (x:1) (Z / E)
+    // An identity type indicates that the corresponding analysis was used
+    // to confirm the identity of the reported product.
+    IDENTITY = 2;
+    // Yields should only be reported as quantitative yields, e.g., measured
+    // using an isolated weight analysis, quantitative NMR with an internal
+    // standard, LC peak area with an internal standard and calibration curve
+    // obtained using an authentic standard. 
+    YIELD = 3;
+    // Selectivites include enantiomeric excess/ratios, diastereomic ratios,
+    // etc. The detailed type should be specified in selectivity_type.
+    SELECTIVITY = 4;
+    // The purity of the product mixture, normalized to all other species
+    // observed in the analysis.
+    PURITY = 5;
+    // Integrated peak area, e.g., as in an LC chromatogram.
+    AREA = 6;
+    // Retention time, e.g., as in an LC or GC chromatogram.
+    RETENTION_TIME = 7;
+    // Raw counts, e.g., as in an EIC or TIC from a MS analysis. If an EIC,
+    // the m/z ratio(s) should be defined in the details.
+    COUNTS = 8;
+    // Intensity, e.g., of a TLC spot, ELSD, or GC FID.
+    INTENSITY = 9;
+    // Direct m/z measurement, e.g., from HRMS. The value field should be used
+    // to report the m/z, not the "mass" field.
+    MZ = 10;
+    // Temperature, e.g., from a melting point experiment.
+    TEMPERATURE = 11;
+    // Angle, e.g., from a circular dichroism experiment. Units of degrees
+    // centimer squared per decimol are assumed.
```

Comment:
We should probably add an Angle message with units to be consistent with e.g. Temperature.

Reply:
Will add

Reply:
With the move of analytical data even for simple measurements like CD, MP, or HRMS back to the analysis message, I don't believe this suggestion is relevant

Reply:
No longer relevant